recipe/meta.yaml

```
@@ -28,13 +28,12 @@ source:
   - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
     sha256: {{ sha256 }}
     patches:
-      - fix_cub_constexpr.diff  # [win64]
       - support_new_cuda_layout.diff  # [((linux or win64) and (cuda_compiler_version or "").startswith("12")) or (build_platform != target_platform)]
 
 build:
   number: {{ number }}
   # For CUDA 12, the compiler version is hard-coded in the cuda120.yaml migrator
-  skip: true  # [(py < 39) or cuda_compiler_version not in ("11.8", "12.4")]
+  skip: true  # [(py < 39) or cuda_compiler_version not in ("11.8", "12.5")]
```

CUDA 12.5 stops supporting ppc64le. Let us follow CUDA and see if anyone comes to complain. (CuPy has no CI running on ppc64le anyway, it was already a long stretch to support ppc64le this long.)
```suggestion
  skip: true  # [(py < 39) or cuda_compiler_version not in ("11.8", "12.5") or ppc64le]
```